deploy/olm-catalog/ibm-block-csi-operator-community/1.11.2/manifests/ibm-block-csi-operator.v1.11.2.clusterserviceversion.yaml

```diff
@@ -0,0 +1,672 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: ibm-block-csi-operator.v1.11.2
+  namespace: placeholder
+  labels:
+    operatorframework.io/arch.amd64: supported
+    operatorframework.io/arch.s390x: supported
+    operatorframework.io/arch.ppc64le: supported
+  annotations:
+    capabilities: "Seamless Upgrades"
+    categories: "Storage,Cloud Provider"
+    certified: "true"
+    containerImage: quay.io/ibmcsiblock/ibm-block-csi-operator:1.11.2
```

Comment:
the community points to quay.io for the operator and the non-community to redhat registry?

Reply:
This is what happens also at 1.11.0 and 1.12.0 (see: gg "containerImage:").

Reply:
ack, that's what i thought